store/cassandra/store_cassandra.go

```diff
@@ -190,11 +190,15 @@ func NewCassandraStore(addrs, keyspace, consistency, CaPath, Username, Password,
 	}
 	cluster.Consistency = gocql.ParseConsistency(consistency)
 	cluster.Timeout = time.Duration(timeout) * time.Millisecond
+	cluster.ConnectTimeout = cluster.Timeout
 	cluster.NumConns = writers
 	cluster.ProtoVersion = protoVer
+	cluster.DisableInitialHostLookup = disableInitialHostLookup
 	var err error
 	tmpSession, err := cluster.CreateSession()
 	if err != nil {
+		log.Info("cassandra_store: session timeout: %s", cluster.Timeout.String())
```

Comment:
we generally don't log config params elsewhere, so i don't think we should do it for this value either